src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import bigLogo from "../../assets/20220206_152352.png";
+import logo from "../../assets/20220203_122320.png";
+
+import { Link } from "react-router-dom";
+
+const Navbar = () => {
+  const [navLinksVisible, setNavLinksVisible] = useState(false);
+
+  return (
+    <nav className="navigation flex-row space-between align-center">
+      <div className="left-navigation flex-row align-center">
+        <i
+          onClick={() => setNavLinksVisible(!navLinksVisible)}
+          className="fas fa-bars hamburger"
+          id="hamburger"
+        ></i>
+
+        <Link to={"/"}>
+          <img src={bigLogo} className="big-logo" alt="logo" />
+        </Link>
+        <Link to={"/"}>
```

Comment:
I don't think {} is needed, in the <Link> tag. You can simply do this .. to="/" and it will work.

Reply:
Done